test/images/hostexec/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.2
+
+run apk --update add curl netcat-openbsd iproute2 && rm -rf /var/cache/apk/*
+
+CMD [ "sleep", "3600" ]
```

Comment:
how did we come up with this time? if the intent is to block until killed can we a bit more explicit? e.g.
```sh
rm -f /fifo && mkfifo /fifo && exec cat </fifo
```